From a88e50e9c9f9a0f0dad33d2e5129a5b2a0a16dcb Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@apache.org>
Date: Thu, 29 Jun 2017 16:27:50 -0700
Subject: [PATCH 2551/2848] HADOOP-14024. KMS JMX endpoint throws
 ClassNotFoundException. Contributed by John
 Zhuge.

(cherry picked from commit 1272ee153b1a1578abf822b2141d8336d9c5652f)
(cherry picked from commit 2956c584c30feeda995ead493ef8d3992f659424)

Change-Id: I7694a1b2d202db2cb76bc2fdd36bda649e889fd7
---
 hadoop-common-project/hadoop-kms/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-kms/pom.xml b/hadoop-common-project/hadoop-kms/pom.xml
index 2d9960e..c3b94ce 100644
--- a/hadoop-common-project/hadoop-kms/pom.xml
+++ b/hadoop-common-project/hadoop-kms/pom.xml
@@ -84,7 +84,7 @@
     <dependency>
       <groupId>org.mortbay.jetty</groupId>
       <artifactId>jetty</artifactId>
-      <scope>test</scope>
+      <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-- 
1.7.9.5

